tools/README.md

```diff
@@ -8,7 +8,11 @@
 
 ## Vim plugins
 
-- ...
+- [syntastic](https://github.com/scrooloose/syntastic) - syntax checking plugin
```

Comment:
what do you use instead? i haven't switched to neovim because of the plugins i use :(